src/main/java/tutorlink/commandpackage/DeleteAssignmentCommand.java

```diff
@@ -29,8 +28,7 @@ public DeleteAssignmentCommand(String matricNumner, String courseID, String assi
     @Override
     public CommandResult execute() {
         try {
-            CourseList courseList = students.getStudent(this.matricNumber).courses;
-            AssignmentList assignmentList = courseList.getCourseByID(this.courseID).getAssignmentList();
+            AssignmentList assignmentList = Command.getAssignmentList();
```

Comment:
DeleteAssignmentCommand should have access to AssignmentList since it inherits Command